src/Symfony/Bundle/WebServerBundle/WebServerConfig.php

```diff
@@ -28,8 +28,8 @@ public function __construct($documentRoot, $env, $address = null, $router = null
             throw new \InvalidArgumentException(sprintf('The document root directory "%s" does not exist.', $documentRoot));
         }
 
-        if (null === $file = $this->guessFrontController($documentRoot, $env)) {
-            throw new \InvalidArgumentException(sprintf('Unable to guess the front controller under "%s".', $documentRoot));
+        if (null === $file = $this->findFrontController($documentRoot, $env)) {
+            throw new \InvalidArgumentException(sprintf('Unable to find the front controller under "%s" (expected to find it as: app_%2$s.php, app.php, index_%2$s.php, index.php).', $documentRoot, $env)));
```

Comment:
What about moving the logic to determine the possible alternatives into a dedicated method to mitigate the risk of inconsistencies between the `findFrontController()` method and the error message being displayed?